pipelines/batch/src/main/java/com/google/fhir/analytics/FhirSearchUtil.java

```diff
@@ -317,6 +332,88 @@ Bundle searchByPatientAndLastDate(
     return query.execute();
   }
 
+  /**
+   * Fetch the details of bulk export job for the given bulkExportStatusUrl
+   *
+   * @param bulkExportStatusUrl the url of the bulk export job
+   * @return BulkExportHttpResponse - the status and details of the bulk export job
+   * @throws IOException
+   */
+  public BulkExportHttpResponse fetchBulkExportHttpResponse(String bulkExportStatusUrl)
+      throws IOException {
+    IHttpResponse httpResponse = fetchUtil.fetchResponseForUrl(bulkExportStatusUrl);
+    BulkExportHttpResponse bulkExportHttpResponse = new BulkExportHttpResponse();
+    bulkExportHttpResponse.setHttpStatus(httpResponse.getStatus());
+    if (httpResponse.getHeaders("expires") != null
```

Comment:
nit: consider adding string constants for these headers (here and everywhere else that it is used more than once).